bin/node/runtime/src/lib.rs

```diff
@@ -503,6 +500,7 @@ parameter_types! {
 	pub const SlashDeferDuration: pallet_staking::EraIndex = 2;
 	pub const RewardCurve: &'static PiecewiseLinear<'static> = &REWARD_CURVE;
 	pub const MaxNominatorRewardedPerValidator: u32 = 256;
+	pub const OffendingValidatorsThreshold: Perbill = Perbill::from_percent(17);
```

Comment:
What is this?

Reply:
[Moved from DisabledValidatorsThreshold variable ](https://github.com/paritytech/substrate/pull/9448/files#diff-67684005af418e25ff88c2ae5b520f0c040371f1d817e03a3652e76b9485224aL463)